parsons/google/google_sheets.py

```diff
@@ -177,6 +177,8 @@ def append_to_sheet(self, spreadsheet_id, table, sheet_index=0):
                 Parsons table
             sheet_index: int (optional)
                 The index of the desired worksheet
+            user_entered_value: bool (optional)
+                If True, will submit cell values as entered (required for entering formulas)
```

Comment:
Could you be a bit more descriptive as to what things look like when this is a set to False? Like what is the output in the GoogleSheet?